### ISSUE TITLE
About accuracy of training from scratch on ImageNet-1k

### ISSUE THREAD

**Issue by jackroos** [google-research/vision_transformer#50]
Hi,

Thanks for your great work! I have several questions about reproducing the ImageNet-1k experiments:

1. I am wondering what's the top-1 accuracy of ViT-B-32 (without fine-tuning on 384x384 resolution) while training scratch from ImageNet-1k. In the paper, the table5 shows the fine-tuning accuracy with 384x384 resolution is 73.38.

2. Do you observe an overfitting phenomenon in this setting? Or do you observe a much higher training top-1 accuracy (e.g., >90%) than validation? If so, do you use some additional regularization methods to mitigate it?
